### DISCUSSION TITLE
zh-cn/posts/2024/09/19/publish-faster/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#79] [Comments]
# zh-cn/posts/2024/09/19/publish-faster/

笔记软件里面的草稿越来越多，但是发布的文章却越来越少了。造成这个情况的原因有：用于写博客的时间少了、对文章的要求过高。囤着草稿是不好的，写好的文章应该发出来，为了帮助自己和他人。
刚写博客那会，我对文章内容没有太高要求，就是想着写好后几天内改几次（吐槽：这要求还不高吗），力求通俗易懂。后来写作要求提升了两次。

https://cyrusyip.org/zh-cn/posts/2024/09/19/publish-faster/

**Comment by residualsun1** [comment]
我作为读者是期待看到更新的，如果博主囤着稿子太久，每次点进来我也会有一种“扑了个空”的感觉哈哈。

**Reply by CyrusYip** [reply]
哈哈，那你可以订阅 RSS，不用经常去看。

**Reply by residualsun1** [reply]
RSS好像会把网站的一些设置、样式抽离掉。虽说内容更重要，但我还是觉得文章和网站不应该分离，不然我会觉得不完整。而且我也比较喜欢点进网站偶然发现更新的感觉。

**Reply by CyrusYip** [reply]
> RSS好像会把网站的一些设置、样式抽离掉。

确实有这个问题，所以我只用 RSS 阅读器来订阅更新，看文章的时候还是跳转到对应的网页。

**Reply by residualsun1** [reply]
好办法😄。

**Comment by hongtaoh** [comment]
没必要完美主义。我一般都是开始写了就写完，然后发出去，不满意的话之后慢慢再改。
